### PR TITLE
HEC-1046: update CT status and README

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,39 @@
 
 # hec-stubs
 
-This is a placeholder README.md for a new repository
+# Test Data
```
Comment (AndyHWChung) [inline README.md:4]: I thought it'd be handy to document everything that's easy to see in one place as soon you go to this stub repo in github. With this collected info in this README I was thinking we could probably do without the commented out docs in the controllers and other places describing the same thing - what do you think @priyankadhiman3006  @meghana-viswanath? Save us updating the docs in multiple places if we need to change something in the future.

File: app/uk/gov/hmrc/hecstubs/models/accountOverviewDetails/ReturnStatus.scala [hunk 2]
```diff
@@ -24,10 +24,12 @@ sealed trait ReturnStatus extends Product with Serializable {
...
-  case object ReturnFound extends ReturnStatus { override def value = "Return Found" }
-  case object NoticeToFileIssued extends ReturnStatus { override def value = "Notice to File Issued" }
-  case object NoReturnFound extends ReturnStatus { override def value = "No Return Found" }
-  case object NoAccountingPeriodFound extends ReturnStatus { override def value = "No Accounting Period Found" }
+  case object ReturnFound extends ReturnStatus { override def value = "1" }
+
+  case object NoticeToFileIssued extends ReturnStatus { override def value = "2" }
+
+  case object NoReturnFound extends ReturnStatus { override def value = "3" }
+
```
Comment (meghana-viswanath) [inline app/uk/gov/hmrc/hecstubs/models/accountOverviewDetails/ReturnStatus.scala:32]: do we not need `NoAccountingPeriodFound `?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/hecstubs/models/accountOverviewDetails/ReturnStatus.scala:32]: not anymore - that information is now returned as status `2` under the outer status in the CT response

**Review by meghana-viswanath — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (priyankadhiman3006) [reply README.md:4]: good idea 👍

File: app/uk/gov/hmrc/hecstubs/models/CompanyAccountingPeriodResponse.scala [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  val error404Response: JsValue =
+    Json.parse(s"""
+       |{
+       |   "code": "NOT_FOUND",
+       |   "reason": " The remote endpoint has indicated that no data can be found for the given identifier."
+       |}
+       |""".stripMargin)
+
+  val error400Response: JsValue =
+    Json.parse(
```
Comment (priyankadhiman3006) [inline app/uk/gov/hmrc/hecstubs/models/CompanyAccountingPeriodResponse.scala:105]: Should we include a scenario where we have invalid date format, either start date or end date?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/hecstubs/models/CompanyAccountingPeriodResponse.scala:105]: yeah, I can add that in - shouldn't be too difficult to add that :+1:

Comment (AndyHWChung) [reply app/uk/gov/hmrc/hecstubs/models/CompanyAccountingPeriodResponse.scala:109]: updated now - I noticed that I was missing a few test cases in the unit tests as well so I added them in

**Review by priyankadhiman3006 — APPROVED** [review]

**Pull request merged**